include/deal.II/lac/solver_gmres.h

```diff
@@ -664,12 +710,14 @@ class SolverFGMRES : public SolverBase<VectorType>
     /**
      * Constructor. By default, set the maximum basis size to 30.
      */
-    explicit AdditionalData(const unsigned int max_basis_size = 30,
+    explicit AdditionalData(const unsigned int max_basis_size         = 30,
+                            const bool use_truncated_mpgmres_strategy = true,
                             const LinearAlgebra::OrthogonalizationStrategy
                               orthogonalization_strategy =
                                 LinearAlgebra::OrthogonalizationStrategy::
                                   delayed_classical_gram_schmidt)
```

Comment:
This change is not backward compatible. To be backward compatible, you need to add the `use_truncated_...` argument at the end.

Reply:
If you do this, it would probably be useful to also put the new member variable at the bottom of the class.

Reply:
This `AdditionalData` object is for the `MPGMRES` class which we introduce. So there is no issue with backwards compatibility.

Do you want to have the boolean to go last?

Reply:
Let's just keep the same order between the two classes. So yes, let's put the `bool` last.